x-pack/plugins/security_solution/public/common/components/events_viewer/events_viewer.test.tsx

```diff
@@ -15,29 +15,36 @@ import { mockEventViewerResponse } from './mock';
 import { StatefulEventsViewer } from '.';
 import { defaultHeaders } from './default_headers';
 import { useFetchIndexPatterns } from '../../../detections/containers/detection_engine/rules/fetch_index_patterns';
-import { mockBrowserFields } from '../../containers/source/mock';
+import { mockBrowserFields, mockDocValueFields } from '../../containers/source/mock';
 import { eventsDefaultModel } from './default_model';
 import { useMountAppended } from '../../utils/use_mount_appended';
 
+jest.mock('../../components/url_state/normalize_time_range.ts');
+
 const mockUseFetchIndexPatterns: jest.Mock = useFetchIndexPatterns as jest.Mock;
 jest.mock('../../../detections/containers/detection_engine/rules/fetch_index_patterns');
-mockUseFetchIndexPatterns.mockImplementation(() => [
-  {
-    browserFields: mockBrowserFields,
-    indexPatterns: mockIndexPattern,
-  },
-]);
 
 const mockUseResizeObserver: jest.Mock = useResizeObserver as jest.Mock;
 jest.mock('use-resize-observer/polyfilled');
 mockUseResizeObserver.mockImplementation(() => ({}));
 
-const from = 1566943856794;
-const to = 1566857456791;
+const from = '2019-08-26T22:10:56.791Z';
+const to = '2019-08-27T22:10:56.794Z';
 
 describe('EventsViewer', () => {
   const mount = useMountAppended();
 
+  beforeEach(() => {
+    mockUseFetchIndexPatterns.mockImplementation(() => [
+      {
+        browserFields: mockBrowserFields,
```

Comment:
very nit - Won't block you on it, but since the first three properties here repeat throughout the file, I would set a 
```
const defaultMocks =  {
        browserFields: mockBrowserFields,
        indexPatterns: mockIndexPattern,
        docValueFields: mockDocValueFields,
        isLoading: false,
      }
```
and spread {...defaultMocks} in the objects below so it was clear that only one property is changing